packages/foreman/rubygem-fog-vsphere/rubygem-fog-vsphere.spec

```diff
@@ -88,6 +88,9 @@ cp -pa .%{gem_dir}/* \
 %{gem_instdir}/tests
```

Comment:
Looks like these are no longer part of the gem so this line should be dropped

Reply:
Yeah, you are right. Any other place I should delete them from?